crates/router/src/connector/adyen/transformers.rs

```diff
@@ -2040,6 +2052,188 @@ impl From<AdyenNotificationRequestItemWH> for AdyenResponse {
             refusal_reason: None,
             refusal_reason_code: None,
             additional_data: None,
+            auth_code: None,
+        }
+    }
+}
+
+// Payouts
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenPayoutResponse {
+    psp_reference: String,
+    result_code: AdyenStatus,
+    amount: Option<Amount>,
+    merchant_reference: Option<String>,
+    refusal_reason: Option<String>,
+    refusal_reason_code: Option<String>,
+    additional_data: Option<AdditionalData>,
+    auth_code: Option<String>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenPayoutEligibilityRequest {
+    pub amount: Amount,
+    pub merchant_account: String,
+    pub payment_method: PayoutCardDetails,
+    pub reference: String,
+    pub shopper_reference: String,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct PayoutCardDetails {
+    #[serde(rename = "type")]
+    pub _type: String,
+    pub number: String,
+    pub expiry_month: String,
+    pub expiry_year: String,
+    pub holder_name: String,
+}
+
+#[derive(Clone, Default, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub enum PayoutEligibility {
+    Y,
+    #[default]
+    N,
+    D,
+    U,
+}
+
+// Payouts eligibility request transform
+impl<F> TryFrom<&types::PayoutsRouterData<F>> for AdyenPayoutEligibilityRequest {
+    type Error = Error;
+    fn try_from(item: &types::PayoutsRouterData<F>) -> Result<Self, Self::Error> {
+        let auth_type = AdyenAuthType::try_from(&item.connector_auth_type)?;
+        let payout_method_data = get_payout_card_details(&item.request.payout_method_data).map_or(
+            Err(errors::ConnectorError::MissingRequiredField {
+                field_name: "payout_method_data",
+            }),
+            Ok,
+        )?;
```

Comment:
Add `get_payout_method_data` in connector/utils.rs so that other can reuse it
```suggestion
        let payout_method_data = get_payout_card_details(&item.request.get_payout_method_data()?);
```